content/ko/docs/topics/charts.md

```diff
@@ -4,241 +4,234 @@ description: "차트 형식을 설명하고, 헬름으로 차트를 빌드하기
-to mark a chart as deprecated. If the **latest** version of a chart in the
-repository is marked as deprecated, then the chart as a whole is considered to
-be deprecated. The chart name can be later reused by publishing a newer version
-that is not marked as deprecated. The workflow for deprecating charts, as
-followed by the [kubernetes/charts](https://github.com/helm/charts) project is:
+차트 저장소에서 차트를 관리할때, 가끔 차트를 deprecate하는 것이 필요하다.
+`Chart.yaml` 에 있는 선택 필드인 `deprecated` 는 차트가 미사용 예정임을
+표시하는데 사용될 수 있다. 저장소에 있는 차트의 **최신** 버전이 미사용 예정으로
+표시된다면, 차트는 전체가 미사용 예정이라고 판단한다. 미사용예정을 체크하지 않은
+새로운 버전을 발행함으로써 차트명을 이후에 재사용할수 있다. 차트를 미사용 하는 것에 대한 [kubernetes/charts](https://github.com/helm/charts) 프로젝트를 따르는 작업흐름은 다음과 같다.
 
-1. Update chart's `Chart.yaml` to mark the chart as deprecated, bumping the
-   version
-2. Release the new chart version in the Chart Repository
-3. Remove the chart from the source repository (e.g. git)
+1. 차트의  `Chart.yaml`를 업데이트 하여 차트를 미사용 예정으로 표시하여 버전을 올린다.
+2. 차트 저장소에 새로운 차트 버전을 Release 한다
+3. 차트를 소스 저장소에서 지운다 (예. git)
 
 ### 차트 타입
 
-The `type` field defines the type of chart. There are two types: `application`
-and `library`. Application is the default type and it is the standard chart
-which can be operated on fully. The [library chart]({{< ref
-"/docs/topics/library_charts.md" >}}) provides utilities or functions for the
-chart builder. A library chart differs from an application chart because it is
-not installable and usually doesn't contain any resource objects.
+`type` 필드는 차트의 타입을 정의한다. `application`, `library` 의
+두가지 타입이 있다. application은 기본 타입이며
+완전히 작동할 수 있는 표준 차트이다
+[library chart]({{< ref "/docs/topics/library_charts.md" >}}) 는 차트 빌더에
+유틸리티나 함수를 제공한다. library chart는 설치 불가능하고, 보통
+어떤 리소스 오브젝트도 가지지 않는다는 것이 application chart와 다르다.
 
-**Note:** An application chart can be used as a library chart. This is enabled
-by setting the type to `library`. The chart will then be rendered as a library
-chart where all utilities and functions can be leveraged. All resource objects
-of the chart will not be rendered.
+**참고:** application chart는 library chart로 사용될수 있다. 타입을
+`library`로 셋팅함으로써 가능하다. 차트는 모든 유틸리티와
+함수 기능을 활용할수 있는 상태의 library chart로 렌더링된다. 모든 차트의
+리소스 오브젝트는 렌더링되지 않는다.
 
 ## 차트 라이센스, README 와 NOTES
 
-Charts can also contain files that describe the installation, configuration,
-usage and license of a chart.
+차트는 설치, 환경설정, 사용법, 차트의 라이센스를
+설명하는 파일을 가질 수 있다.
 
-A LICENSE is a plain text file containing the
-[license](https://en.wikipedia.org/wiki/Software_license) for the chart. The
-chart can contain a license as it may have programming logic in the templates
-and would therefore not be configuration only. There can also be separate
-license(s) for the application installed by the chart, if required.
+LICENSE는 차트에 대한 [license](https://en.wikipedia.org/wiki/Software_license) 를 포함하는
+일반 텍스트 파일이다. 차트는 템플릿 안의 프로그래밍
+로직을 가지는 것으로써의 라이센스를 포함할 수 있으므로 환경 설정
+전용이 아니다. 또한, 필요하다면 차트에 의해
+설치된 어플리키이션에 대한 라이센스는 나눠질 수 있다.
```

Comment:
```suggestion
LICENSE는 차트에 대한 [license](https://en.wikipedia.org/wiki/Software_license) 를 
포함하는 일반 텍스트 파일이다. 차트는 템플릿 안에서 프로그래밍 로직으로 
사용될 수도 있고, 이에 설정요소가 될 수 없는 라이센스를 
포함할 수 있다. 필요한 경우, 차트에 의해 설치된 어플리케이션을 위해 
별도의 라이센스로 나눠질 수 있다.
```
표현을 조금 바꾸어보았습니다. 확인 후 반영 부탁드립니다.

Reply:
다른 부분은 반영 완료했지만, 한 문장은 제가 잘 이해하지 못한 것 같습니다 😂
- 원문 : The chart can contain a license as it may have programming logic in the templates
and would therefore not be configuration only.
- 변경 전 : 차트는 템플릿 안의 프로그래밍 로직을 가지는 것으로써의 라이센스를 포함할 수 있으므로 환경 설정 전용이 아니다.
- 리뷰 주신 내용 : 차트는 템플릿 안에서 프로그래밍 로직으로  사용될 수도 있고, 이에 설정요소가 될 수 없는 라이센스를 포함할 수 있다.

원문에 따르면, '설정 용도로만 사용되는 것이 아니다'로 이해됩니다. (제가 잘못 이해한것일까요? 😭)
의견 부탁드립니다 🙏
@bluefriday

Reply:
@sunminjeon 해당 부분에 대하여 의견 부탁드립니다.